packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue

```diff
@@ -56,13 +56,13 @@ $breakpoints: (medium: 520px, large: 960px);
     &:nth-child(1) {
         display: block;
     }
-    @include media('>medium') {
-        margin: 0 spacing(d) 0 0;
+    @include f.media('>520px') {
```

Comment:
How come these have changed?

Reply:
Good spot. I actually have no idea how these have changed, i'm certain I didn't make this change. Will revert.

Reply:
Right I remember now. Further up in this file there's new values overriding the `$breakpoints` from within `fozzie` it was causing errors so I've just passed them in directly instead here.